src/cartogram_info/read_csv.cpp

```diff
@@ -46,36 +46,36 @@ int extract_area_col_index(
 
 int extract_inset_col_index(
   const csv::CSVReader &reader,
-  const argparse::ArgumentParser &arguments)
+  const Arguments &args)
 {
   // Find index of column with inset specifiers. If no inset column header was
   // passed with the command-line flag --inset, the header is assumed to be
   // "Inset". This default value is set in parse_arguments.cpp.
-  auto inset_header = arguments.get<std::string>("-I");
+  auto inset_header = args.inset_col;
   int inset_col = reader.index_of(inset_header);
 
   return inset_col;
 }
 
 int extract_color_col_index(
   const csv::CSVReader &reader,
-  const argparse::ArgumentParser &arguments)
+  const Arguments &args)
 {
   // Find index of column with color specifiers. If no color column header was
   // passed with the command-line flag --color, the header is assumed to be
   // "Color".
-  auto color_header = arguments.get<std::string>("-C");
+  auto color_header = args.color_col;
```

Comment:
This is not sufficient to account for the British spelling "Colour". Over here, we would need to use the CSV reader to our advantage to fiugre out whether "Colour" or "Color" exist as headers, and have that be the `color_header`. Another idea to discuss here would be if we can make this search case-insensitive.